components/Quote/Quote.tsx

```diff
@@ -46,14 +45,13 @@ export const Quote = ({
 }: QuoteProps) => (
   <AnimateInView animation={animation} delay={delay}>
     <Container
-      as="article"
       width="full"
       pt={addDarkOverlay && !isMinimal ? 6 : undefined}
       pb={addDarkOverlay && !isMinimal ? 4 : undefined}
       className={cnb(styles.root(isMinimal, addDarkOverlay, quoteOnRight, !!barColor), className)}
       {...props}
     >
-      <div className={cnb(styles.content(!!barColor, quoteOnRight), accentBorderColors[barColor])}>
+      <blockquote className={cnb(styles.content(!!barColor, quoteOnRight), accentBorderColors[barColor])}>
```

Comment:
For the `Quote` component, use blockquote element on the wrapper level after seeing the way it's used.